build/charts/antrea/crds/tier.yaml

```diff
@@ -8,6 +8,33 @@ spec:
   group: crd.antrea.io
   versions:
     - name: v1alpha1
+      served: true
+      storage: false
+      deprecated: true
```

Comment:
According to @antoninbas 's suggestion, please remove this, we may submit in a separate PR if we decide to support from K8s 1.19.
https://github.com/antrea-io/antrea/pull/5108#discussion_r1240547356

Reply:
Done.